randovania/games/prime1/logic_database/header.json

```diff
@@ -1439,6 +1439,16 @@
                         },
                         "lock": null
                     },
+                    "Power Door": {
```

Comment:
Power Door would imply that canonically, Power Beam is the *minimum* requirement to open such a door. Future maintainers of this file may get this confused with vanilla blue doors.

Also, Power Door could be confused with the "Powered" doors in crashed frigate etc.

Please use `Power Beam Only` for any and all code/UI which references this new door